test/pgw_proxy_SUITE.erl

```diff
@@ -934,6 +942,45 @@ create_session_overload_response(Config) ->
     ok.
 
 
+%%--------------------------------------------------------------------
+path_maint() ->
+    [{doc, "Check that Echo requests are sent"}].
+path_maint(Config) ->
```

Comment:
Could it be worth factoring out the similarities between this test and the one in ggsn_proxy_SUITE.erl  ?

Reply:
That would apply to all the protocol handler tests. Yes, it could be done, but it would be a lot of work. Certainly more that can go into this PR